docs/blog/2020-11-05-zmk-sotf-3.md

```diff
@@ -0,0 +1,156 @@
+
+#### Deep Sleep
+
+[petejohanson] has contributed the initial [deep sleep](https://github.com/zmkfirmware/zmk/pull/211) support to ZMK. This work also
+included some automatic power savings by switching to PORT events on the nrf52 chips, which reduces the idle power draw, even without deep sleep. Deep sleep is currently not turned on by default, but will be soon.
+
+### Miscellaneous
+
+#### Output Selection
+
+[joelspadin] added [output selection](/docs/behavior/outputs) to allow selecting whether to send output over USB or BLE if both are connected. This should now help avoiding having "double keypresses" when your keyboard is plugged into a host.
+
+#### Bootloader Corruption Fix
+
+[Nicell] has already [blogged about this](/blog/2020/10/03/bootloader-fix), but for those that missed it, a _major_, and incredibly difficult to pin down bug involving corruption of the bootloader on devices using the Adafruit nrf52 bootloader has been fixed by [Nicell] himself. If you've encountered this bug, flashing the latest firmware should prevent it from reoccurring. Unfortunately, due to the nature of this fix, you will need to re-pair your keyboard with your hosts, as the fix involves changing where settings are stored in the flash of the controller.
+
+#### Official USB Product ID
+
+[petejohanson] has gotten an official [USB product ID](https://github.com/openmoko/openmoko-usb-oui/pull/15) assigned to the ZMK Firmware. For anyone looking to uniquely identify a USB device running the ZMK Firmware, you can match on:
+
+- Vendor ID: `0x1d50`
+- Product ID: `0x615e`
+
+We are incredibly grateful that Openmoko Inc., in the wake of discontinuing the openmoko projects, has made this an option for OSS projects.
+
+#### Development: Remote Docker Container Integration
+
+[idan](https://github.com/idan) contributed [VSCode devcontainer integration](https://github.com/zmkfirmware/zmk/pull/209) to make it easier for developers to build and develop ZMK without having to do complicated local toolchain setup and configuration. This also opens up some amazing future flexibility for things like [GitHub Codespaces](https://twitter.com/ZMKFirmware/status/1315760616779505678?s=20).
+
+There's some follow up tweaks necessary for better supporting using this with user config repositories, which will be available soon.
+
+## New Shields
+
+- Reviung41 in [#297](https://github.com/zmkfirmware/zmk/pull/297) - [Nicell]
+- Boardsource 3x4 in [#296](https://github.com/zmkfirmware/zmk/pull/296) - [neegool](https://github.com/neegool)
+- NIBBLE in [#292](https://github.com/zmkfirmware/zmk/pull/292) - [jaygreco](https://github.com/jaygreco)
+- Microdox in [#245](https://github.com/zmkfirmware/zmk/pull/245) - [careyk007](https://github.com/careyk007)
+- MakerDiary M60 in [#233](https://github.com/zmkfirmware/zmk/pull/233) - [megamind4089]
```

Comment:
> [megamind4089]

Throughout the post, some names have links, but some don't?

Reply:
Alternate format for links in Markdown. These use the links at the end of the MD file.

Reply:
This saves me from repeating the same link a bunch of times